JSON.md

```diff
@@ -0,0 +1,53 @@
+Bencodex JSON Representation
+===========================
+
+## Introduction
+
+Bencodex JSON Representation is a standard way to represent a given Bencodex value to valid JSON text to be interchanged.
+
+This specification defines a standard way to represent a Bencodex value to (a subset of) JSON and how to decode/encode them.
+
+### Conventions Used in This Document
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14[^RFC-2119] when, and only when, they appear in all capitals, as shown here.
+
+## Prefixes
+
+Bencodex JSON Representation defines a set of prefixes **SHALL** be used for JSON[^JSON] string value to denote a corresponding Bencodex type. The defined prefix are following:
+
+- `0x` denotes a hexadecimal byte-string. The rest of the string **SHOULD** be parsed as a byte-string encoded in hexadecimal.
+  - The casing of the value **MAY** be ignored.
+- `U+FEFF BYTE ORDER MARK` denotes a string. The rest of the string **MUST** be parsed as a string.
+- `b64:` denotes a base64-encoded byte-string. The rest of the string **MUST** be parsed as a byte-string encoded in base64 as defined in [RFC 4648](https://www.rfc-editor.org/rfc/rfc4648.html)[^RFC-4648].
```

Comment:
It's trivial, but it would be better if we use consistent terms/spellings between the entire specification (see also *README.md*).

```suggestion
- `0x` denotes a hexadecimal byte string. The rest of the string **SHOULD** be parsed as a byte string encoded in hexadecimal.
  - The casing of the value **MAY** be ignored.
- `U+FEFF BYTE ORDER MARK` denotes a Unicode string. The rest of the string **MUST** be parsed as a Unicode string.
- `b64:` denotes a base64-encoded byte string. The rest of the string **MUST** be parsed as a byte string encoded in base64 as defined in [RFC 4648](https://www.rfc-editor.org/rfc/rfc4648.html)[^RFC-4648].
```